Userland/testing_libc.cpp

```diff
@@ -0,0 +1,8 @@
+#include <netdb.h>
+int main() 
+{
+    getservent();
+    getservent();
+    getservent();
+    getservent();
+}
```

Comment:
This seems more like a test utility to have on your local machine than something we should include in the userland utilities :)